website/install.html

```diff
@@ -100,6 +100,19 @@
       .os-column-table > .windows {
         grid-area: windows;
       }
+
+    @media (prefers-color-scheme: dark) {
+      a:link {color:#268BD2;}
+      a:visited {color:#6C71C4;}
+      body {background-color:  #002B36; color: white;}
+      td {color:black;}
+      .macos-logo-color {filter: invert(85%);} 
```

Comment:
Instead of creating a class, could we select based on the `src` attribute? For example, `.os-logo[src="macos.svg"]`

Reply:
I ended up doing this therefore not needing class macos-logo.